_pages/documentation.md

```diff
@@ -0,0 +1,74 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: documentation
+permalink: "/documentation"
+page_class: p-documentation
+
+# NO EDITING ABOVE THIS LINE
+# # # # # #
+
+seo_description: "GeoTrellis is an open source project, so contributions of any kind are welcome and appreciated! Contributors will need to sign a CLA (Contributor’s License Agreement). These details and others relating to GeoTrellis contributions can be found here."
+title: "Documentation"
+
+# # # # # #
+# Page copy
+
+# intro section
+intro: 
+  title: Documentation
+  blurb: In addition to our API documentation, we provide a detailed [User Guide.](https://docs.geotrellis.io/en/latest/) Please see that guide for setup instructions, tutorials, and explanations of GeoTrellis/GIS concepts.
+  body: If you can’t find exactly what you’re looking for in either our Guide or API docs, feel free to join us in our [Gitter channel,](https://gitter.im/geotrellis/geotrellis) where we can answer your questions live.
+  button:
+    text: Go to docs
+    url: https://geotrellis.readthedocs.io/en/latest/
+# intro section
+
+# user guide section
+user-guide:
+  title: GeoTrellis User Guide
+  blurb: Our [User Guide](https://geotrellis.readthedocs.io/en/latest/) is hosted on ReadTheDocs, and provides all the conceptual information required to use GeoTrellis.
+  links:
+    - text: Home
+      url: https://docs.geotrellis.io 
+    - text: Setup
+      url: https://docs.geotrellis.io/en/latest/tutorials/setup.html 
+    - text: Quick Start
+      url: https://docs.geotrellis.io/en/latest/tutorials/quickstart.html 
+    - text: Core Concepts
+      url: https://docs.geotrellis.io/en/latest/guide/core-concepts.html 
+
+# user guide section
+
+# scala api section
+scala-api:
+  title: Scala API Documentation
+  blurb: The latest Scaladoc tracks the master branch on github. Tagged releases are identified by their version number.
+  links:
+    - text: Latest 
+      url: https://geotrellis.github.io/scaladocs/latest/#geotrellis.package
+    - text: Version 1.1
+      url: https://geotrellis.github.io/scaladocs/1.1/#geotrellis.package
+    - text: Version 1.0
+      url: https://geotrellis.github.io/scaladocs/1.0/#geotrellis.package
+    - text: Version 0.10
+      url: https://geotrellis.github.io/scaladocs/0.10/#geotrellis.package
+    - text: Version 0.9
+      url: https://geotrellis.github.io/scaladocs/0.9/#geotrellis.package
+    - text: Version 0.8
+      url: https://geotrellis.github.io/scaladocs/0.8/#geotrellis.package
+    - text: Version 0.7
+      url: https://geotrellis.github.io/scaladocs/0.7/#geotrellis.package
+    - text: Version 0.6
+      url: https://geotrellis.github.io/scaladocs/0.6/#geotrellis.package
```

Comment:
Perhaps we can add a link to the github branch which corresponds to the API docs